@ecomplus/storefront-components/src/js/LoginBlock.js

```diff
@@ -103,7 +103,8 @@ export default {
     submitLogin () {
       if (!this.isWaitingLogin) {
         this.isWaitingLogin = true
-        const { email, docNumber } = this
+        const { docNumber } = this
+        const email = this.email.toLowerCase()
```

Comment:
`this.email` pode ser undefined, daria um fatal aqui

Reply:
Como que seria undefined, se a função é executada no login? Porque pra logar, a primeira chave de identificação é o e-mail e não consegue submeter o login, sem colocar ele

Reply:
o `this.email` inicia com o valor de uma prop não requerida
https://github.com/ecomplus/storefront/blob/02ce3ef0862bb9c3133ce6b2fcb29fc716024554/%40ecomplus/storefront-components/src/js/LoginBlock.js#L37
https://github.com/ecomplus/storefront/blob/02ce3ef0862bb9c3133ce6b2fcb29fc716024554/%40ecomplus/storefront-components/src/js/LoginBlock.js#L52
então se o método for chamado internamente não necessariamente o `email` vai estar setado

Reply:
se ele for chamado só no submit do form não vai dar esse problema a não ser que o usuário remova o required no html e isso não deve acontecer, eu não tinha visto mesmo onde ele é chamado mas de qualquer forma você tá partindo de uma premissa ruim aqui